Clean Swift/Models.xctemplate/___FILEBASENAME___Models.swift

```diff
@@ -21,13 +21,18 @@ enum ___VARIABLE_sceneName___ {
         let type: RequestType
     }
 
-    struct Response {
-        var rawData: AnyObject?
-        var error: Error?
-    }
+    struct TableModel {
```

Comment:
Why TableModel? Shouldn't it be more abstract? I mean - not all controllers use TableView...